src/commands/android/subcommands/install/system-image.ts

```diff
@@ -0,0 +1,123 @@
+import colors from 'ansi-colors';
+import inquirer from 'inquirer';
+
+import Logger from '../../../../logger';
+import {APILevelNames} from '../../constants';
+import {Platform} from '../../interfaces';
+import {getBinaryLocation} from '../../utils/common';
+import {execBinarySync, spawnCommandSync} from '../../utils/sdk';
+import {showMissingBinaryHelp} from '../common';
+import {AvailableSystemImages} from '../interfaces';
+
+export async function installSystemImage(sdkRoot: string, platform: Platform): Promise<boolean> {
+  try {
+    const sdkmanagerLocation = getBinaryLocation(sdkRoot, platform, 'sdkmanager', true);
+    if (!sdkmanagerLocation) {
+      showMissingBinaryHelp('sdkmanager');
+
+      return false;
+    }
+
+    const stdout = execBinarySync(sdkmanagerLocation, 'sdkmanager', platform, '--list | grep "system-images;"');
```

Comment:
`grep` might not work on Windows system as it is not installed there by default.